dot_config/nvim/lua/tap/plugins/init.lua

```diff
@@ -140,6 +140,7 @@ return require('packer').startup(function(use)
     -- native neovim LSP support
     use {
         'neovim/nvim-lspconfig', -- LSP server config
+        after = "nvim-notify",
```

Comment:
Perhaps need to look at loading nvim-notify ahead of everything to ensure it's configured as anything can use the `vim.notify` API, but this will do for now